tmt/steps/provision/podman.py

```diff
@@ -140,7 +140,7 @@ def start(self):
         self.debug(f"Start container '{self.image}'.")
         self.container_id = self.podman(
             ['run'] + workaround +
-            ['--name', self.container, '-v', f'{workdir}:{workdir}:Z',
+            ['--name', self.container, '-v', f'{workdir}:{workdir}:z',
```

Comment:
Good catch!

Reply:
Fortunately `podman` has these two nicely described in the man page.